ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -20,11 +20,22 @@ import { ThresholdOptions } from '../../model/thresholds';
  */
 export interface TimeSeriesChartOptions {
   queries: TimeSeriesQueryDefinition[];
-  show_legend?: boolean;
+  legend: LegendOptions;
   unit?: UnitOptions;
   thresholds?: ThresholdOptions;
 }
 
+export const DEFAULT_LEGEND: LegendOptions = {
+  show: false,
+  position: 'bottom',
+};
```

Comment:
One possible way to make this more compact (less JSON! 🎉 ): remove the `show` boolean and make `legend` optional. Basically you can infer `show` based on whether you have a legend options object like the `position`. (This assumes there's no point in keeping legend options around if it's not being shown.)

Reply:
Yeah this makes sense! Just updated it to remove `show` and make legend optional--the only downside I see is when you're editing legend options, if you hide and then unhide the legend, you lose the previous value that `position` was set to. So if `position` was initially 'right', it changes to 'bottom' from DEFAULT_LEGEND when legend is toggled off.